From d304124529a803cd073586a48983216173a8ce77 Mon Sep 17 00:00:00 2001
From: "Kothapeta, BikshapathiX" <bikshapathix.kothapeta@intel.com>
Date: Thu, 14 Jul 2022 09:03:47 +0530
Subject: [PATCH 1/2] Revert OAM-102765 because of conflicts

Revert "point host camera service to latest fix"

This reverts commit 2a3b25bfca1b7b14314fdc158f759641a61f1890.

Signed-off-by: Kothapeta, BikshapathiX <bikshapathix.kothapeta@intel.com>

diff --git a/scripts/setup_host.sh b/scripts/setup_host.sh
index 22da8d6..6252625 100755
--- a/scripts/setup_host.sh
+++ b/scripts/setup_host.sh
@@ -119,7 +119,7 @@ function install_host_service() {
     cd host_camera
     git clone https://github.com/projectceladon/host-camera-server.git
     cd host-camera-server
-    git checkout b715695dc2e47eaf2a7d275093419394efe0a7c3
+    git checkout ba2707587e8f3ad03714d5e3ed47844e83cde7c0
     mkdir build
     cd build
     cmake ..
-- 
2.37.1

